app/views/assignments/index.html.erb

```diff
@@ -37,12 +37,18 @@
     <%end%>
       
       <td>
-        <%= link_to "Edit Week", edit_assignment_path(week) %>
+        <% if user_signed_in? %>
+          <%= link_to "Edit Week", edit_assignment_path(week) %>
+        <% end %>
       </td>
   </tr>
 <% end %>
 <tr>
-  <%= link_to "Add Week", new_week_path %>
+  <td>
+    <% if user_signed_in? %>
+      <%= link_to "Add Week", new_week_path %>
+    <% end %>
```

Comment:
This is an authorization concern which is coming next. We'll use something like `if policy(Week.new).new?`.